File: java/maven/1_hello_world/src/main/java/com/example/Main.java
package com.example;

import static spark.Spark.*;

public class Main {

    /**
     * Метод для запуска WebSocket
     *
     * @param args
     */
    public static void main(String[] args) {

        /**
         * Запрос GET /hello
         * Доступен по ссылке http://localhost:4567/hello
         * или по IP http://127.0.0.1:4567/hello
         */
        get("/hello", (req, res) -> "Hello World");
    }
}

File: kotlin/maven/1_hello_world/src/main/kotlin/com/example/Main.kt
package com.example

import spark.kotlin.*

/**
 * Метод для запуска WebSocket
 */
fun main() {

    /**
     * Запрос GET /hello
     * Доступен по ссылке http://localhost:4567/hello
     * или по IP http://127.0.0.1:4567/hello
     */
    get("/hello") {
        "Hello World"
    }
}